src/party.cpp

```diff
@@ -465,7 +466,7 @@ void Party::clearPlayerPoints(Player* player)
 
 bool Party::canOpenCorpse(uint32_t ownerId) const
 {
-	if (Player* player = g_game.getPlayerByID(ownerId)) {
+	if (std::shared_ptr<Player> player = g_game.getPlayerByID(ownerId)) {
```

Comment:
Could this be `const auto`
```suggestion
	if (const auto player = g_game.getPlayerByID(ownerId)) {
```